packages/@sanity/form-builder/src/inputs/arrays/ArrayOfPrimitivesInput/ArrayOfPrimitivesInput.tsx

```diff
@@ -131,6 +131,10 @@ export class ArrayOfPrimitivesInput extends React.PureComponent<Props> {
     }
   }
 
+  handleFocusRoot = () => {
+    this.props.onFocus([])
```

Comment:
Should `[]` be a constant here?

Reply:
Very marginal gains, since it's not likely to be used directly in rendering / equality checking anywhere.